### ISSUE TITLE
502 Gateway Error

### ISSUE THREAD

**Issue by gvjensen** [tiliado/nuvolaruntime#568]
Hi - not sure why this is happening, but whenever I fire up the applet (i.e. for Amazon Music), I am getting a 502 gateway error (from Sparky Linux / Debian 10).

Thoughts - Glenn
glenn.jensen@alumni.ucsd.edu

**Comment by fenryxo** [comment]
Hello @gvjensen. Could you attach a screenshot of Nuvola window with that error?

**Comment by fenryxo** [comment]
I think I've found the source of the error. Could you verify that it has been fixed?

**Comment by gvjensen** [comment]
You found it... working fine now...

Currently running SparkLinux - a Debian 10 variant.  Thanks a bunch!

**Comment by gvjensen** [comment]
> Hello @gvjensen. Could you attach a screenshot of Nuvola window with that error?

Sorry... missed your email.. the issue has been fixed.  The symptom was that the app would act like it was starting up, and then simply hang.... (and I saw the gateway error when I fired it up from the terminal).

Ironically, my Linux Mint environment worked fine, so it had to be something with Debian 10 / SparkyLinux.  I went through all the normal troubleshooting steps... super happy!

Works great now... not sure how it was fixed.
